### ISSUE TITLE
How to effectively use multi-core parallelism on high performance computers？

### ISSUE THREAD

**Issue by lovechang1986** [autogluon/autogluon#2083]
I want to use autogluon on a high-performance machine based on the BSUB job scheduling system. Although I set up the CPU in fit and applied for 10 cores through the BSUB command, the algorithm only used a single core, based on the results of system monitoring. So how do I properly use multi-core parallelism on high-performance computers?

Also, autogluon basically only uses a single core when it comes to cross-node parallel jobs, so the second question is how do I execute the AUTOGLUON algorithm in parallel across nodes?

**Comment by Innixma** [comment]
Closing issue as it is too broad and with too few details to efficiently answer. Please create a new issue and provide code examples to a specific use-case you would like to accelerate.
